### DISCUSSION TITLE
请求在仿真中给机器人添加相机和雷达

### DISCUSSION THREAD

**Discussion by Huisouan** [Genesis-Embodied-AI/genesis-world#315] [Q&A]
Genesis是一个非常好的rl工具，不过目前貌似还无法使用诸如雷达或者相机一类的视觉的传感器。请问官方在未来有相关的计划吗？

**Comment by zhouxian** [comment]
相机已经有了呀 雷达我们会很快加上的

**Reply by Huisouan** [reply]
非常感谢

**Reply by NowLoadY** [reply]
> 相机已经有了呀 雷达我们会很快加上的
> […](#)
> On Thu, Dec 26, 2024 at 10:53 AM Huisouan ***@***.***> wrote: Genesis是一个非常好的rl工具，不过目前貌似还无法使用诸如雷达或者相机一类的视觉的传感器。请问官方在未来有相关的计划吗？ — Reply to this email directly, view it on GitHub <#315>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AEV4V6OKPSSZR7PCMXDQ2VD2HNVT5AVCNFSM6AAAAABUGS4NWCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXG42DIMRTGM> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

怎么获取对应相机的frame_buffer阿
我没去翻文档，在这里直接问是可以的吗）

**Reply by zhouxian** [reply]
what exactly do you mean frame_buffer?
The api is like this `rgb, depth, seg_mask, normap_map = cam.render()`

**Comment by XXLiu-HNU** [comment]
为什么感觉这里边的深度图有点不太正确
<img width="1358" height="815" alt="6af903d6-3ae9-4df6-9eaa-4bfed413b899" src="https://github.com/user-attachments/assets/2d86dc99-cff7-483f-a30f-9cc90b99ac22" />
